Update broad -> popgen links (#2)
populationgenomics/seqr: README.md

seqr
====
![Unit Tests](https://github.com/populationgenomics/seqr/workflows/Unit%20Tests/badge.svg?branch=master) | ![Local Install Tests](https://github.com/populationgenomics/seqr/workflows/local%20install%20tests/badge.svg?branch=master)

seqr is a web-based tool for rare disease genomics.

> This repository contains code that underlies the [Population Genomics seqr instance](http://seqr.populationgenomics.org.au) and other seqr deployments.

Reviewer: ```suggestion
This repository contains code that underlies the [CPG seqr instance](http://seqr.populationgenomics.org.au).
```
